src/utils/updateUser.ts

```diff
@@ -17,3 +17,7 @@ export const updateUser = async (user: Partial<User>) => {
   }
   return res;
 };
+
+export async function getServerSideProps() {
```

Comment:
won't be able to execute getServerSideProps in a util, is only allowed in a page